process-document/src/test/kotlin/com/ritense/processdocument/service/DocumentDelegateServiceTest.kt

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.processdocument.service
+
+import com.ritense.document.domain.impl.*
```

Comment:
I see some * imports here, these should be fully written out

Reply:
Yes, that's what IntelliJ does while 'Organizing imports' :-( I will fix this.

Reply:
Can you please change the settings in IntelliJ: `Editor > Code Style > Java/Kotlin > Imports` and fix this?